HealthcareSystem/Backend/Service/DrugConsumptionService/DrugConsumptionService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Backend.Model.Pharmacies;
+using Backend.Repository.DrugConsumptionRepository;
+
+namespace Backend.Service.DrugConsumptionService
+{
+    public class DrugConsumptionService : IDrugConsumptionService
+    {
+        private readonly IDrugConsumptionRepository _drugConsumptionRepository;
+
+        public DrugConsumptionService(IDrugConsumptionRepository drugConsumptionRepository)
+        {
+            _drugConsumptionRepository = drugConsumptionRepository;
+        }
+        public List<DrugConsumption> GetAllDrugConsumptions()
+        {
+            return _drugConsumptionRepository.GetAllDrugConsumptions().ToList();
+        }
+
+        public DrugConsumption GetDrugConsumptionById(int id)
+        {
+            return _drugConsumptionRepository.GetDrugConsumptionById(id);
+        }
+
+        public void CreateDrugConsumption(DrugConsumption dc)
+        {
+            _drugConsumptionRepository.CreateDrugConsumption(dc);
+        }
+
+        public void UpdateDrugConsumption(DrugConsumption dc)
+        {
+            _drugConsumptionRepository.UpdateDrugConsumption(dc);
+        }
+
+        public void DeleteDrugConsumption(DrugConsumption dc)
+        {
+            _drugConsumptionRepository.DeleteDrugConsumption(dc);
+        }
```

Comment:
Is there really a use case that requires these to be modifiable? Again, I feel like they should be historical records - you lower the quantity of a drug, and make a DrugConsumption to record that it happened. It's not something I see ever being updated or deleted (except maybe by DB admins after a certain amount of time when we no longer need the history). 
Even if such a use case exists though, I definitely not your current one, so I probably wouldn't have added these methods either.

Reply:
Fixed in 6498babbb3b08e9824f980dab9090775e0555339.